### DISCUSSION TITLE
Securing BIND 9 with AppArmor/Firejail/SecompBPF

### DISCUSSION THREAD

**Discussion by netblue30** [netblue30/firejail#4649] [Announcements]
Nice video from ISC, the makers of Bind DNS server:

https://www.youtube.com/watch?v=Cez-RkSQEHY

In other news, I moved the video channel from youtube to Brighteon (preppers and vitamin sellers). The way youtube is going with censorship and mass surveillance, they will become a liability for us. The new guys are in Texas, but I can see the video traffic coming from Cloudflare servers. There is a chance they are properly mirrored all over the world. If you have problems outside North America let me know, and will try to find another place.

Channel here: https://www.brighteon.com/channels/netblue30

**Comment by reinerh** [comment]
> In other news, I moved the video channel from youtube to Brighteon (preppers and vitamin sellers). The way youtube is going with censorship and mass surveillance, they will become a liability for us. The new guys are in Texas, but I can see the video traffic coming from Cloudflare servers. There is a chance they are properly mirrored all over the world. If you have problems outside North America let me know, and will try to find another place.
> 
> Channel here: https://www.brighteon.com/channels/netblue30

I don't care much where the videos are hosted, but still I found this choice a bit odd.
The page appears a bit dubious, and is bugging you to subscribe to their newsletter...

In my opinion hosting it on a free software focused peertube instance would have been more appropriate (e.g. https://peertube.debian.social/).

**Comment by netblue30** [comment]
> I don't care much where the videos are hosted, but still I found this choice a bit odd.

Yes, they are overly aggressive.  When I opened the account I had to disable the adblocker! But other than that, they seem to be doing fine technically. Unlike youtube they have no visibility. I estimate about 20 views per day, not more than half a gig bandwidth per day. That's quite small by any standards.

Peertube is a good idea, I'll look into it - maybe not exactly debian.social, it is more for conferences and Debian project videos. Anyway, I'll be mainly collecting external videos, reviews and such, in any possible language.
